lm_diagnoser/extractors/base_extractor.py

```diff
@@ -50,23 +62,31 @@ class Extractor:
     init_embs : FullActivationDict
         initial embeddings that are loaded from file or set to zero.
     """
-    def __init__(self, config: Dict[str, Any]) -> None:
-        self.config = config
+    def __init__(self, model: str, vocab: str, corpus: LabeledCorpus,
+            load_modules: str = '', 
+            activations: List[ActivationName] = [('hx', 1), ('cx', 1)],
+            output_dir: str = '',
```

Comment:
That's strange, since "None" is not a string, right? Wouldn't I break anything by changing this?

Reply:
Ah the reason I said this was because in the docstring the type was said to be `Optional[str]`. I see now that that doesn't comply with the type here, but I think it's best if we adhere to `Optional[str]` with `None` as default.